solutions/bronze/cf-1808B.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: cf-1808B
+source: CF 1808B
+title: Playing in a Casino
+author: Amit Singh Bisht
+---
+
+## Explanation
+
+Essentially, the solution is to calculate the difference between the sum of the larger elements and the sum of the smaller
+elements in each column, and weighting this difference by the number of times that the smaller element appears in that column.
+This is based on the observation that if we have n numbers, and we want to minimize the sum of absolute differences between
+each number and some target value, then the target value should be the median of those n numbers. This solution applies this
+observation to each column of the matrix, separately.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2 M)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	ios_base::sync_with_stdio(false);
+	cin.tie(NULL);
+	int t;
+	cin >> t;
+	for (int i = 0; i < t; i++) {
+		int n, m;
+		cin >> n >> m;
+		vector<vector<double>> a(n, vector<double>(m)),
+		    pre(n, vector<double>(m));
```

Comment:
move `pre`'s declaration down to where it's actually used

Reply:
wait `pre` isn't even used what

Reply:
I was previously thinking about using pre, but I forgot to remove it once I realized I didn't need it.